geppetto-ui/src/components/big-image-viewer/BigImageViewer.js

```diff
@@ -1,121 +1,155 @@
-import React, {Component} from 'react';
-import {withStyles} from '@material-ui/core';
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core';
 import OpenSeaDragon from 'openseadragon';
-import * as util from "../../utilities";
-import { faHome, faSearchPlus, faSearchMinus } from "@fortawesome/free-solid-svg-icons";
-import CustomToolbar from "../common/CustomToolbar";
+import PropTypes from 'prop-types';
+import * as util from '../../utilities';
+import {
+  faHome,
+  faSearchPlus,
+  faSearchMinus,
+} from '@fortawesome/free-solid-svg-icons';
+import CustomToolbar from '../common/CustomToolbar';
 
 const styles = {
-    bigImageViewer: {
-        height: '100%',
-    },
+  bigImageViewer: {
+    height: '90%',
```

Comment:
This can be improved as height setting won't work unless the parent has fixed height. With flex it's easy to avoid this. Also, if want some padding (90% rather than 100% I suppose this is for a padding), better use a padding.